app/src/main/java/org/jellyfin/androidtv/ui/livetv/LiveTvGuideActivity.java

```diff
@@ -142,20 +142,23 @@ public void onClick(View v) {
                 showFilterOptions();
             }
         });
+        findViewById(R.id.filterButton).setContentDescription(getString(R.string.lbl_filters));
```

Comment:
`findViewById` is an expensive operation so it's better to assign the found view to a variable before using it to avoid finding a view multiple times. (same for the next 2)